### ISSUE TITLE
how can i print a receipt with 4 columns

### ISSUE THREAD

**Issue by fxia86** [kakzaki/blue_thermal_printer#94]
like this.
```
NAME     PRICE      QTY     TOTAL
apple     2.8        3       8.4
banana    1.6        6       9.6
...
```

**Comment by enricoirawan** [comment]
same here, i want to make a receipt with 4 column, but i cant

**Comment by kakzaki** [comment]
i hope it will be resolved at 1.1.6

**Comment by Alecckie** [comment]
@kakzaki , 
Kindly provide more information on how to use printNcolumns because it is not working correctly and the documentation on format is not clear.

<img width="236" alt="Screenshot 2022-06-14 211409" src="https://user-images.githubusercontent.com/10174852/173660361-70db0b81-0233-4347-aee9-b10cc0962a54.png">
